ext/pdo_dblib/tests/pdodblib_001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+PdoDblib basic
+--EXTENSIONS--
+pdo_dblib
+--SKIPIF--
+<?php
+require __DIR__ . '/config.inc';
+getDbConnection();
+?>
+--FILE--
+<?php
+
+require __DIR__ . '/config.inc';
+$db = getDbConnection(PdoDblib::class);
+
+$db->query("create table #pdo_dblib_001(name varchar(32)); ");
+$db->query("insert into #pdo_dblib_001 values('PHP'), ('PHP6');");
```

Comment:
```suggestion
$db->query("CREATE TABLE #pdo_dblib_001(name varchar(32)); ");
$db->query("INSERT INTO #pdo_dblib_001 values('PHP'), ('PHP8');");
```
Also is `#` a valid character for SQL stuff in general?

Reply:
At least mysql may recognize it as a comment.

Reply:
Yes, ext/dblib uses `#` characters before table names. I'm not sure what the exact reason is, but this seems quite legit :)

Reply:
Adding # in front of the table name means a session temporary table in SQL Server (two # means global temporary table). Maybe you are trying to use a temporary table?

Reply:
Thanks for the explanation, I'm not very familiar with sql server. I'm not really trying to achieve anything specific with temporary tables, I was just using the syntax the other tests do. :)

Reply:
I see now, thank you.

Many other places also use temporary tables, so I think either one is fine here for now. It might be a good idea to fix it all together later.